src/auth/ui/vue/authForms/manageBuilderId.vue

```diff
@@ -77,10 +79,23 @@ export default defineComponent({
     methods: {
         async startSignIn() {
             this.stage = 'WAITING_ON_USER'
-            this.error = await this.state.startBuilderIdSetup()
-            if (this.error) {
+            client.startAuthFormInteraction(this.state.featureType, 'awsId')
+            const authError = await this.state.startBuilderIdSetup()
+
+            if (authError) {
+                this.error = authError.text
                 this.stage = await this.state.stage()
+
+                client.failedAuthAttempt({
```

Comment:
For future work I would suggest avoiding this "round-tripping" pattern by moving more logic/state out of the webview. Shorter data flow paths are easier to understand.

Reply:
I agree, if we want to invest more in to webviews it'll be useful to add functionality to be able to use multiple WebviewClients so we don't need to cram everything in to one `VueWebview` class